lib/suggestion-list.js

```diff
@@ -245,31 +253,49 @@ export default class SuggestionList {
       }
     } else {
       this.destroyOverlay()
-      this.activeEditor = editor
-      this.displayBufferPosition = bufferPosition
-      const marker = this.suggestionMarker = editor.markBufferRange([bufferPosition, bufferPosition])
-      this.overlayDecoration = editor.decorateMarker(marker, {type: 'overlay', item: this, position: 'tail'})
-      this.addBindings(editor)
+      if (editor) {
```

Comment:
Not really, because we need to call `destroyOverlay` if the editor is not the same. I guess we could have an early return that calls `destroyOverlay`, but I'm not sure if that would be clearer.